packages/manager/modules/nutanix/src/dashboard/nodes/list/add-nodes/add-nodes.routing.js

```diff
@@ -0,0 +1,38 @@
+import NodeExpressOrderLinkGenerator from '../../../node-express-order-link-generator';
+import { getOrderExpressUrl } from '../../../../constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('nutanix.dashboard.nodes.all.add-nodes', {
+    url: '/add-nodes',
+    views: {
+      modal: {
+        component: 'addNutanixNodesModal',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      goBack: /* @ngInject */ ($state) => () =>
+        $state.go('nutanix.dashboard.nodes.all'),
+      nodeTechnicalDetails: /* @ngInject */ (NutanixService, cluster) =>
+        NutanixService.getNodeHardwareInfo(cluster.targetSpec.nodes[0].server),
+      nodePricing: /* ngInject */ (clusterAddOns) => {
+        return clusterAddOns[0].billing.pricing;
+      },
+      handleError: /* @ngInject */ (Alerter, goBack) => (error) => {
+        Alerter.error(
+          error.message || error.data?.message,
+          'nutanix_dashboard_alert',
+        );
+        goBack();
+      },
+      nodeOrderLinkGenerator: /* @ngInject */ (serviceName, clusterAddOns) =>
+        new NodeExpressOrderLinkGenerator(serviceName, clusterAddOns[0]),
+      expressOrderLink: /* @ngInject */ (coreConfig) =>
+        getOrderExpressUrl(coreConfig.getUser().ovhSubsidiary),
+      handleSuccess: /* @ngInject */ (Alerter, goBack) => (message) => {
+        Alerter.success(message, 'nutanix_dashboard_alert');
+        goBack();
+      },
```

Comment:
Except `goBack`, is it possible to define the other dependencies in the parent state to use them both in dashboard and listing part ?